zk-sdk/src/encryption/pedersen.rs

```diff
@@ -19,13 +19,12 @@ use {
     zeroize::Zeroize,
 };
 
-lazy_static::lazy_static! {
-    /// Pedersen base point for encoding messages to be committed.
-    pub static ref G: RistrettoPoint = RISTRETTO_BASEPOINT_POINT;
-    /// Pedersen base point for encoding the commitment openings.
-    pub static ref H: RistrettoPoint =
-        RistrettoPoint::hash_from_bytes::<Sha3_512>(RISTRETTO_BASEPOINT_COMPRESSED.as_bytes());
-}
+/// Pedersen base point for encoding messages to be committed.
+pub const G: RistrettoPoint = RISTRETTO_BASEPOINT_POINT;
+/// Pedersen base point for encoding the commitment openings.
+pub static H: std::sync::LazyLock<RistrettoPoint> = std::sync::LazyLock::new(|| {
+    RistrettoPoint::hash_from_bytes::<Sha3_512>(RISTRETTO_BASEPOINT_COMPRESSED.as_bytes())
+});
```

Comment:
@samkim-crypto @joncinque - Do you have any concerns about this change from a compatibility point of view ? Namely, you can see later in the diff that stuff like `*G` is now just `G`

Reply:
Per https://github.com/anza-xyz/agave/pull/6057#issuecomment-2874386243, Jon had no issues so marking this resolved